projects/playground/src/app/app.component.spec.ts

```diff
@@ -213,9 +210,9 @@ describe('AppComponent', () => {
         'Access Token'
       );
       const form = component.accessTokenOptionsForm.controls;
-      form.usePopup.setValue(false);
-      form.ignoreCache.setValue(false);
-      (authMock.getAccessTokenSilently as jasmine.Spy).and.returnValue(
+      form['usePopup'].setValue(false);
+      form['ignoreCache'].setValue(false);
+      ((authMock.getAccessTokenSilently as unknown) as jest.SpyInstance).mockReturnValue(
```

Comment:
We can't cast to jest.SpyInstance without casting to unknown first. This could be approved upon in a future PR if needed.